<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="horizontalWidget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QWidget" name="verticalWidget_2" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>10</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QPushButton" name="cerrarSesion">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(85, 85, 255);</string>
              </property>
              <property name="text">
               <string>Cerrar Sesión</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="horizontalWidget_2" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>700</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/pickachu.png);</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4"/>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>170</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="verticalWidget_2" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>250</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QGraphicsView" name="graphicsView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/partida-rapida.jpg);</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="partidaRapida">
              <property name="font">
               <font>
                <pointsize>17</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 255, 0);</string>
              </property>
              <property name="text">
               <string>Partida Rápida</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="verticalWidget_3" native="true">
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>250</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QGraphicsView" name="graphicsView_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/ladder.jpg);</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ladder">
              <property name="font">
               <font>
                <pointsize>17</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Ladder</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="verticalWidget_4" native="true">
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>250</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QGraphicsView" name="graphicsView_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/build.jpg);</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="team">
              <property name="font">
               <font>
                <pointsize>17</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 0, 255);
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Team Builder</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="archivo.qrc"/>
 </resources>
 <connections/>
</ui>
